tasks.go

```diff
@@ -125,24 +126,20 @@ func buildHooks(yamlData TsuruYaml, envs []bind.EnvVar) error {
 	return execScript(cmds, envs, os.Stdout)
 }
 
-func readProcfile() (string, error) {
-	procfilePath := fmt.Sprintf("%s/%s", defaultWorkingDir, "Procfile")
-	f, err := filesystem().Open(procfilePath)
+func readProcfile(path string) (string, error) {
+	f, err := filesystem().Open(fmt.Sprintf("%v/Procfile", path))
 	if err != nil {
 		return "", err
 	}
 	defer f.Close()
 	procfile, err := ioutil.ReadAll(f)
-	if err != nil {
```

Comment:
We should not remove the error handling code here.

Reply:
:up: